pkgs/development/libraries/libavif/default.nix

```diff
@@ -45,6 +55,23 @@ stdenv.mkDerivation rec {
     libyuv
   ];
 
+  postPatch = ''
+    substituteInPlace contrib/gdk-pixbuf/avif.thumbnailer.in \
+      --replace '@CMAKE_INSTALL_FULL_BINDIR@/gdk-pixbuf-thumbnailer' "$out/bin/avif-thumbnailer"
```

Comment:
Maybe something like `$out/libexec/gdk-pixbuf-thumbnailer-with-avif` so it does not pollute `PATH` when accidentally installed. And so that the relationship to `gdk-pixbuf-thumbnailer` is cleaner.

Reply:
Should we do the same for `webp-pixbuf-loader`?

Reply:
I guess that would be a breaking change